packages/common/src/ui/layout/tables/components/Cells/CurrencyInputCell/CurrencyInputCell.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { CellProps } from '../../../columns';
+import { CurrencyInput } from '../../../../../components/inputs';
+import { DomainObject } from '../../../../../../types';
+import { useDebounceCallback } from '../../../../../../hooks';
+
+type R<T> = T &
+  DomainObject & { update?: (key: string, value: string) => void };
+
+type CP<T> = CellProps<R<T>>;
+
+export const CurrencyInputCell = <T extends DomainObject>({
+  rowData,
+  column,
+}: CP<T>): React.ReactElement<CP<T>> => {
+  const [buffer, setBuffer] = React.useState(Number(column.accessor(rowData)));
+
+  const noop = () => {};
+  const updater = useDebounceCallback(rowData.update ?? noop, [rowData], 250);
+
+  return (
+    <CurrencyInput
+      maxWidth={column.width}
+      value={buffer}
+      onChangeNumber={newNumber => {
+        setBuffer(newNumber);
+        updater(String(column.key), String(newNumber));
+      }}
+    />
+  );
```

Comment:
maybe it's preferrable that there is no $ prefix in a cell in a table?

Reply:
I think it looks great - might not suit every client though: Ivory Coast are using `XOF` which takes up a lot of space and I've had to use interesting fonts to get rupees and euros working elsewhere

Reply:
Yeah I did think that XOF would be a bit big. Thinking in the column header might be better 🤔